visual-studio-code-bin/PKGBUILD

```diff
@@ -62,6 +63,10 @@ package() {
   install -Dm 644 "${srcdir}/${_pkg}/resources/completions/zsh/_code" "${pkgdir}/usr/share/zsh/site-functions/_code"
 
   cp -r "${srcdir}/${_pkg}/"* "${pkgdir}/opt/${_pkgname}" -R
-  ln -s /opt/${_pkgname}/bin/code "${pkgdir}"/usr/bin/code
+
+  # Launcher
+  echo "  -> Edit ~/.config/code-flags.conf to add launch flags..."
```

Comment:
Don't do it like this. Instead, create an [.install file](https://wiki.archlinux.org/title/PKGBUILD#install) and do the echo in `post_install` and `post_upgrade`.

Reply:
Done with 7298ec8